readme.md

```diff
@@ -64,10 +64,11 @@ test(t => {
 
 ### Add AVA to your project
 
-Install AVA globally and run it with `--init` to add AVA to your `package.json`:
+Install AVA globally and run it with `--init` to add AVA to your `package.json`.
+We recommend using [yarn](https://yarnpkg.com/) to speed up the installation process.
 
 ```console
-$ npm install --global ava
+$ yarn global add ava
```

Comment:
It should not replace `npm`, but be a recommended alternative.

Like:

We recommend using Yarn bla bla.

yarn install

Alternatively with npm:

npm install